string path
string axis
int64 step_size
---
bool response